src/libs/OptionsListUtils.js

```diff
@@ -587,6 +587,138 @@ function isCurrentUser(userDetails) {
     return _.some(_.keys(loginList), (login) => login.toLowerCase() === userDetailsLogin.toLowerCase());
 }
 
+/**
+ * Build the options for the tree hierarchy via indents
+ *
+ * @param {Object[]} options - an initial strings array
+ * @param {Boolean} options[].enabled - a flag to enable/disable option in a list
+ * @param {String} options[].name - a name of an option
+ * @param {Boolean} [isOneLine] - a flag to determine if text should be one line
+ * @returns {Array<Object>}
+ */
+function getOptionTree(options, isOneLine = false) {
```

Comment:
Will this function be used for anything other than categories? If not, perhaps we could make the function name more specific. Like `getCategoryOptionTree` or something.

Reply:
Agree. Done!